ecosystem/sep-0041.md

```diff
@@ -200,6 +202,12 @@ The event has topics:
 The event has data:
 
 - `i128` the amount transferred.
+- or `map` containing entries:
+  - `amount: i128` the amount minted.
+  - `to_muxed_id: Option<u64 | String | BytesN<32>>` void or absent if no muxed
```

Comment:
Is this implying that `to_muxed_id` is allowed to be present with no value? I feel like a value should be required if there’s a key, and the key itself is the optional part?

Also it’s worth clarifying the data type of the keys here